ports/geckolib/glue.rs

```diff
@@ -2670,12 +2670,11 @@ pub extern "C" fn Servo_DeclarationBlock_SerializeOneValue(
 }
 
 #[no_mangle]
-pub extern "C" fn Servo_SerializeFontValueForCanvas(
+pub unsafe extern "C" fn Servo_SerializeFontValueForCanvas(
```

Comment:
Hmmm... I'm not particularly happy with marking a whole function unsafe just for doing one pointer dereference.

Reply:
It's marked unsafe because it accepts an arbitrary raw pointer as an argument.

Reply:
Hmmm, okay, fair. Then I guess majority of the functions in this file should really be marked unsafe...